src/styles/_class-overrides.scss

```diff
@@ -0,0 +1,525 @@
+    .header__language-nav-LtoR {
+      display: flex;
+      align-items: center;
+      justify-content: flex-start;
+      flex-direction: row;
+      flex-wrap: wrap;
+    }
+    //language button styling
+    .header__language-nav-items {
+      .usa-button--unstyled {
+        @include u-text('accent-cool-lighter');
+        &:hover {
+          @include u-text('primary');
+          @include u-bg('base-dark');
+        }
+      }
+    }
+  }
+}
+
+//desktop view
+@media (min-width: units('desktop')) {
+  //show language selector button and project title
+  .banner__language-selector,
+  .header__title {
+    display: block;
+  }
+  .usa-nav {
+    margin-bottom: units(2);
+  }
+  .header__logo-title {
+    @include flex-center-space-between();
+  }
+  .usa-navbar {
+    .header__logo {
+      width: units('card-lg');
+      height: auto;
+    }
+  }
+
+  .usa-nav {
+    padding: units(2);
+    .usa-nav__inner {
+      .usa-nav__primary-item {
+        .usa-nav__submenu {
+          @include u-bg('base-darker');
+          .usa-nav__submenu-item {
+            @include u-bg('base-darker');
+            a {
+              @include u-text('accent-cool-lighter');
+              @include u-bg('base-darker');
+              @include u-text('normal');
+            }
+          }
+        }
+        //links in nav menu
+        a {
+          @include u-text('disabled-dark');
+          @include u-bg('accent-cool-lighter');
+          @include u-text('bold');
+          padding: units(2);
+          margin: 0;
+          &:hover {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            @include u-text('bold');
+            padding: units(2);
+            margin: 0;
+            border: none;
+          }
+        }
+        .usa-accordion__button[aria-expanded='false'] {
+          @include u-text('disabled-dark');
+          @include u-bg('accent-cool-lighter');
+          @include u-text('bold');
+          padding: units(2);
+          margin: 0;
+          &:hover {
+            @include u-text('base-darker');
+            @include u-bg('accent-cool-lighter');
+            @include u-text('bold');
+            padding: units(2);
+            margin: 0;
+            border: none;
+          }
+          &::after {
+            border: units('2px') solid color('secondary');
+          }
+        }
+        .usa-accordion__button[aria-expanded='true'] {
+          @include u-text('accent-cool-lighter');
+          @include u-bg('base-darker');
+          @include u-text('bold');
+          padding: units(2);
+          margin: 0;
+          &:hover {
+            @include u-text('accent-cool-lighter');
+            @include u-bg('base-darker');
+            @include u-text('bold');
+            padding: units(2);
+            margin: 0;
+            border: none;
+          }
+        }
+      }
+    }
+  }
+
+  /*hide language buttons from nav in desktop view 
```

Comment:
The `mocto-theme.scss` file was renamed, but this shows up as a completely new file, so I can't identify the changes. What changes did you make?

Reply:
Mostly just the media queries - line 325 on. I'm planning to make some functions/mixins to make the code more concise and readable